### ISSUE TITLE
missing code

### ISSUE THREAD

**Issue by Mattosx** [v2v3v4/BigWorld-Engine-2.0.1#1]
baseapp/cellapp/cellappmgr missing all cpp files.

**Comment by cshard447** [comment]
all source code valid

**Comment by cshard447** [comment]
Compile
server files: only centos 5.11 (server)
client files: only windows (visual studio 2008)

**Comment by Mattosx** [comment]
> all source code valid

yeah, the existing source code is valid, but many source codes are missing.

**Comment by loanselot1** [comment]
he is correct, @cshard447 baseapp and cellapp missing its source files, almost so hard to compile cause you have to write them by yourself with looking into symbols in 2010 built object files

**Comment by RevOcelotMGS** [comment]
Do you have the rewritten source code for this? would be great
